api/test/fixtures/rest_framework.py

```diff
@@ -13,3 +16,25 @@ def request_factory() -> APIRequestFactory():
     request_factory = APIRequestFactory(defaults={"REMOTE_ADDR": "192.0.2.1"})
 
     return request_factory
+
+
+@pytest.fixture
+def access_token():
+    token = AccessTokenFactory.create()
+    token.application.verified = True
+    token.application.save()
+    return token
+
+
+@pytest.fixture
+def authed_request(access_token, request_factory):
+    request = request_factory.get(
+        "/", HTTP_AUTHORIZATION=f"Bearer {access_token.token}"
+    )
+
+    return APIView().initialize_request(request)
```

Comment:
Fixtures in this module are just moved around, but I've also fixed an issue in this `authed_request` fixture where it was causing `request.auth` to be a token string, rather than the actual authenticated access token object that it would be in normal circumstances. It was using `force_authenticate` which causes `auth` to be just a string, and to bypass the authentication class. We don't need to do that, we have a real token, so we can just rely on the normal authentication configuration. That's a good thing! It means we're integrating the whole authentication flow into these tests and ensuring it works in all scenarios that we have tests for.